.gitignore

```diff
@@ -17,3 +17,5 @@ javadoc/
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
+
+javadoc/
```

Comment:
I'm not sure if we want to merge this or not...  I'll talk to Conrad and the TAs and let you know.